dnf-testing.sh

```diff
@@ -121,6 +122,7 @@ build()
     if [ ${#output[@]} -eq 3 ] && \
        [ "${output[0]}" = "Successfully" ] && 
        [ "${output[1]}" = "built" ]; then
+        echo "${output[2]}" > container_id.txt
```

Comment:
I would save this into /tmp/container_id.txt. Also if it is not needed afterwards, please, delete the file.